src/tests/ftest/server/daos_server_config.yaml

```diff
@@ -6,6 +6,7 @@ timeout: 130
 server_config:
   name: daos_server
   engines_per_host: 1
+  system_ram_reserved: 16
```

Comment:
can you please explain why there are instances where in the yaml file you set this to 6 and in others 16?

Reply:
The instances where the yaml file value have been reduced to 6 were as a result of the bump in the default value causing a memory check failure. This is likely to be in situations where the environment in which the CI test-in-question is memory constrained and so reserving 8gib rather than 6gib doesn't allow a minimum (4gib) RAM-disk to be allocated to the engine after taking into account other memory reservations in the calculation.

Some other tests in CI that are intermittently experiencing memory check failures (e.g. https://daosio.atlassian.net/browse/DAOS-13918) due to RAM usage spikes have been "fixed" in this PR by increasing the `system_ram_reserved` for hardware based tests to 16gib to give a larger buffer which should reduce if not remove the chance of intermittent failures related to this check.

Reply:
description updated

Reply:
This comment makes me wonder if we have learned that the default should actually be 16 rather than 8. If we're seeing intermittent failures on hardware tests with the default of 8, then it seems likely to me that the default of 8 is probably going to cause failures out in the field.

We knew from the start of this effort that it was going to involve some experimentation to find defaults that strike the right balance between safety and max utilization of the hardware. As such, I think the testing really should use whatever values we're expecting most customers to use. If those values result in lots of intermittent failures in CI, then that is a canary for what to expect in the field.

Reply:
ok, I will increase the default value to 16.